lumen/views/base.py

```diff
@@ -269,7 +268,7 @@ def _validate_pipeline(cls, *args, **kwargs):
     ##################################################################
 
     @classmethod
-    def from_spec(cls, spec: Dict[str, Any], pipeline=None, source=None, filters=None):
+    def from_spec(cls, spec: Dict[str, Any], source=None, filters=None, pipeline=None):
```

Comment:
Should we make it keywords-only argument after spec?

Reply:
Yes, seems like a good idea.

Reply:
Can you file an issue?